snap/bueno_snap.py

```diff
@@ -0,0 +1,270 @@
+#
```

Comment:
This is probably a mistake. Why are SNAP things included in this request? Please remove all SNAP things from this pull request.